### PR TITLE
enabling the setting of some postgresql settings related to autovacuum and checkpointing

### PR DESCRIPTION
**Pull request by tmlink**
Please review @vitroth

### CODE REVIEW

**Comment by tmlink** [comment]
Why am I (  @tmlink ) not getting email for this PR?  I wonder if @vitroth is.

**Comment by tmlink** [comment]
Is anyone getting emails?  @treyharris @philpennock @austinmills @vitroth

**Comment by treyharris** [comment]
I am, but only because I have an explicit filter promoting GitHub messages
that mention my name. In general all GitHub messages skip my inbox because
I couldn’t do anything but file email otherwise.

**Comment by vitroth** [comment]
Yes i'm getting the emails, just busy w/ other items.  github *never* emails you on your own comments, no need to notify you about your own updates...

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.erb ident
```
Comment (vitroth) [inline .gitattributes]: What was the cause for this change?

File: templates/default/postgresql.conf.standard.erb [hunk 2]
```diff
@@ -2,6 +2,8 @@
 # PostgreSQL configuration file
 # -----------------------------
 #
+# @(#) $Id$
```
Comment (treyharris) [inline templates/default/postgresql.conf.standard.erb]: @vitroth The .gitattributes change is to cause this to expand. I'm not sure, do we want to use CVS-style ID's though? They're usually discouraged with git.

**Comment by philpennock** [comment]
The commit message gives no hint that `checkpoint_segments` is being changed, and there's no commentary in the file explaining how the new value was chosen.  So this gives the impression of a cargo-culted number being changed with no reasoning behind it.

Please give the rationale in comments in the file and update the commit message to be more informative.  After that, _then_ LGTM but not as it stands now.

**Comment by tmlink** [comment]
Sorry,  I had put the comments in https://apcera.atlassian.net/browse/SRE-88

$Id$:
It's not really intended to be used here in the RCS/CVS style of giving human readable tags.  With git, it will be expanded to the hex name of the blob.  My intent is to provide an easy at-a-glance way to verify that the file has the expected origin.  (checksums don't work with erb expansion)  Would have saved me a lot of time. 

checkpoint_segments:
pdp is right.  Setting checkpoint_segments on a component database really is cargo cult programming.  Increasing this value is needed in a database with a high volume of updates such as the monitoring database.  But the right setting will vary per database server and be determined based on measurements.  I shouldn't change the global default without a better argument for doing so.  --  slapping own wrist -- will resubmit without this change.

**Comment by tmlink** [comment]
I'm replaying this PR with a much simplier one:   https://github.com/apcera/chef-postgresql/pull/4

**Comment by tmlink** [comment]
I'm replaying this PR with a much simplier one:   https://github.com/apcera/chef-postgresql/pull/4

**Pull request closed without merging**